coral-component-cyclebutton/src/scripts/CycleButton.js

```diff
@@ -62,10 +62,14 @@ class CycleButton extends BaseComponent(HTMLElement) {
   /** @ignore */
   constructor() {
     super();
+
+    if (!this.id) {
+      this.id = commons.getUID();
```

Comment:
This will create the id attribute which will throw an exception
```
Uncaught DOMException: Failed to construct 'CustomElement': The result must not have attributes
```
The ID can be added in the `render()` function